proto/meta.proto

```diff
@@ -529,9 +532,33 @@ message TableParallelism {
   }
 }
 
+// Changes a streaming job in place by
+// overwriting the labels of a materialized view/table/sink with the given id
+message UpdateStreamingJobRequest {
+  // Id of the materialized view, table, or sink which we want to update
+  uint32 id = 1;
+
+  // replace the node_labels map of the streaming job with a given id 
+  // with below map
+  map<string, string> node_labels = 2;
+}
+
+// We do not need to add an explicit status field here, we can just use the RPC status
+message UpdateStreamingJobResponse {}
+
+message GetStreamingJobsRequest {}
+
+// Descriptions of MVs and sinks
+message GetStreamingJobResponse {
+  repeated catalog.Table materialized_views = 1;
+  repeated catalog.Sink sinks = 2;
+}
+
 service ScaleService {
   rpc GetClusterInfo(GetClusterInfoRequest) returns (GetClusterInfoResponse);
   rpc Reschedule(RescheduleRequest) returns (RescheduleResponse);
+  rpc UpdateStreamingJob(UpdateStreamingJobRequest) returns (UpdateStreamingJobResponse); 
```

Comment:
Ditto for the naming.

Reply:
After the controller knows that all new CNs are ready, this RPC should also be issued to meta. So that meta can know and start to do backfilling on theres CN for the new MV. I think this is the missing part of the graph in the PR body.

Reply:
I also thought about that, but disagree. The controller sets the real `node_label` right after starting the new CNs. This way meta can schedule on the new CN immediately after it registered. There is no need to wait for the controller. 

<img width="755" alt="Screenshot 2024-07-22 at 10 45 02" src="https://github.com/user-attachments/assets/83a19d8b-fa8f-4dbe-acf5-3af20e7bd549">

Reply:
In that way meta only knows the label but don't know how many CNs it have to wait until it can do scheduling.